server/src/test/java/io/crate/types/DateTypeTest.java

```diff
@@ -0,0 +1,54 @@
+package io.crate.types;
+
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.nullValue;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DateTypeTest {
+
+    @Test
+    public void test() {
```

Comment:
```suggestion
    public void test_cast_from_invalid_string() {
```

Reply:
I renamed to camelCase style in order to keep it consistent with my other tests. I saw both underscore and camelCase test names and thought both are acceptable. Let me know if underscore is preferred I will update all my test names.

Reply:
We have no strict rule, we just tend to the underscore format for some time now as we think its more readable specially with long test names which we tend to do to express the what-to-test in a better way. But its also fine to use CamelCase here.

Reply:
I will use underscore from now on)